sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareOperation.cs

```diff
@@ -227,5 +249,28 @@ private async ValueTask<Response> UpdateStatusAsync(bool async, CancellationToke
 
             return GetRawResponse();
         }
+
+        /// <summary>
+        /// Cancels a pending or running <see cref="HealthcareOperation"/> from text analytics server.
+        /// </summary>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        public virtual void Cancel(CancellationToken cancellationToken = default)
```

Comment:
I see how KV is not doing it, but to be consistent with our other methods, could you add the scope too?
```
using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(TextAnalyticsClient)}.{nameof(StartCancelHealthJobAsync)}");
scope.Start();
```
etc...